src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/dbf/BuildCacheDBF.java

```diff
@@ -0,0 +1,23 @@
+package com.sonyericsson.jenkins.plugins.bfa.model.dbf;
+
+import com.axis.system.jenkins.plugins.downstream.cache.BuildCache;
+import hudson.Extension;
+import hudson.model.Run;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Gets downstream builds using the build-cache plugin. This should cover most scenarios.
+ *
+ * @author Stephan Pauxberger
+ */
+@Extension(optional = true)
+public class BuildCacheDBF extends DownstreamBuildFinder {
+
```

Comment:
I think this extension will be loaded even if `BuildCache` isn't installed and just fail when calling `getDownstreamBuilds` instead. Because I don't think imports are actually part of the class file when it's loaded.

```suggestion
    static {
        BuildCache.getClass()
    }
```

That static initializer would cause a `ClassNotFoundException` or similar when the extension is loaded, and since it's marked as optional it would just be removed from the extension list during startup instead.